core/src/main/scala/shapeless/datatype/mappable/ToMappable.scala

```diff
@@ -57,18 +57,18 @@ trait LowPriorityToMappableOption0 extends LowPriorityToMappable0 {
   }
 }
 
-trait LowPriorityToMappableSeq0 extends LowPriorityToMappableOption0 {
-  implicit def hconsToMappableSeq0[K <: Symbol, V, H <: HList, T <: HList, M: CanNest, S[_]]
+trait LowPriorityToMappableIterable0 extends LowPriorityToMappableOption0 {
+  implicit def hconsToMappableIterable0[K <: Symbol, V, H <: HList, T <: HList, M: CanNest, S[_] <: Iterable[_]]
```

Comment:
The added subtype constraint is because without it I was seeing ambiguous implicit errors.

Reply:
What was the exact error you were seeing?